.github/workflows/clang-format.yml

```diff
@@ -0,0 +1,27 @@
+name: clang-format
```

Comment:
I think this could be moved to the CI action file like [this](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_id) as it should run at the same time.